app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java

```diff
@@ -131,6 +131,7 @@ public void start(Stage primaryStage) {
     primaryStage.setOnCloseRequest(closeEvent -> {
       if (!AppPreferences.getInstance().isConfirmExit() || mainWindowController.isSaved()) {
```

Comment:
Is it better to reverse the logic of this if statement to gate the dialog and use the `Platform.exit()` on line 157?

Reply:
I did that, and the cancel button stopped working (shuffleboard still exited). Haven't figured that out yet.